trezor-keyring-mv2.js

```diff
@@ -0,0 +1,49 @@
+const { default: TrezorConnect, DEVICE_EVENT } = require('trezor-connect');
+const {
+  TrezorKeyring,
+  KEYRING_TYPE,
+  TREZOR_CONNECT_MANIFEST,
+} = require('./trezor-keyring');
+
+class TrezorKeyringMv2 extends TrezorKeyring {
+  constructor(opts = {}) {
```

Comment:
Is this constructor needed? It appears to do the same thing as the superclass.

Reply:
Not needed, I've removed it now.